docs/docs/installation/upgrade.md

```diff
@@ -24,6 +24,14 @@ If you installed your Keptn instance from the Manifest,
 additional steps are required to use the Helm Chart to upgrade.
 Contact us on Slack for assistance.
 
+## Upgrade to v1beta1 version
+
+If you have previously used Keptn Lifecycle Operator with API
```

Comment:
Do you want to specify the release where this is necessary, as in "When upgrading from Keptn v0.10.x to v0.11.x,..."

Reply:
the v1alpha3 and v1alpha4 are not strictly bind to 0.10.x, so I would keep it more general in this case